README.md

```diff
@@ -168,3 +168,40 @@ The following are the high level steps to update the notices page:
 1. Install [LicensePlist](https://github.com/mono0926/LicensePlist) `brew install licenseplist`
 2. Run `license-plist --markdown-path Acknowledgements.markdown`
 3. Open an issue on [Bedrock](https://github.com/mozilla/bedrock/) with a description and the requested page to update with a link to a document of the generated `Acknowledgements.markdown`
+
+## Releasing
+
+To release a version of Pocket following our every 2 week cycle you should follow the following depending on your release scenario. The following assumes you will be releasing to the Testflight Audience and then after some time promoting that version of the App to the store.
+
+If you need to increment the app version:
+1. If needed pr to increment the app version number in Git on the `develop` branch.
+2. Trigger the `nightly-internal-pipeline` manually to submit a new build to Testflight.
```

Comment:
How is this achieved?

Reply:
Added link to bitrise docs on how to manually trigger a build. https://devcenter.bitrise.io/en/builds/starting-builds/starting-builds-manually.html